models/event.py

```diff
@@ -214,6 +215,23 @@ def rankings(self):
                 self._rankings = None
         return self._rankings
 
+    @property 
+    def rankings_enhanced(self):
+        if self._rankings is not None and self.year == 2016:
+            self._rankings_enhanced = {}
+            team_number_index = self._rankings[0].index("Team")
```

Comment:
use `self.rankings` for these. If `self.rankings` hasn't been called yet, then `self._rankings` will be `None` since it hasn't been lazy loaded yet